gtav/base.lua

```diff
@@ -29,6 +29,11 @@ Citizen.CreateThread(function()
     
     -- FIB lobby: 105.4557, -745.4835, 44.7548
     RequestIpl("FIBlobby")
+
+	--Fib Fountain 174.184,-667.902,43.140
```

Comment:
hey, you might wanna turn this line into:
```lua
-- FIB Fountain 174.184, -667.902, 43.140
```

Reply:
Done :D